tests/ja4-quic-7.0.x-01/test.yaml

```diff
@@ -0,0 +1,19 @@
+requires:
+  min-version: 7.0.6
+  lt-version: 8
+  features:
+    - HAVE_JA4
+
+pcap: ../ja4-quic/input.pcap
+
+args:
+  - -k none
+  - --set app-layer.protocols.tls.ja4-fingerprints=no
```

Comment:
should this be a quic option?

Reply:
It's not needed for the test; will remove